apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateQuestion.tsx

```diff
@@ -67,6 +67,7 @@ export type QuestionFormType = {
     allowFutureDates?: string;
     dataType?: string;
     codeSet: any;
+    mask: string | any;
```

Comment:
Issue: use of `any`

Reply:
faced the same issue as coded type, so I used any there.